assets/js/scripts.js

```diff
@@ -430,6 +432,15 @@ L.LayerGroup.include({
   },
 });
 
+LaIcon = L.Icon.extend({
+    _setIconStyles: function (img, name) {
+        L.Icon.prototype._setIconStyles.call(this, img, name);
+        if (this.options.time && this.options.time !== []) {
+            img.dataset.time = this.options.time
+        }
+    }
+})
+

```

Comment:
Put that in map.js file in MapBase.init() function

Reply:
I'v put it there because of:
https://github.com/jeanropke/RDOMap/blob/master/assets/js/scripts.js#L393

Reply:
Ok I forgot about it
it's fine here

Reply:
in MapBase, I get an error.
Uncaught (in promise) ReferenceError: LaCircle is not defined

Reply:
yeah, left it as it is

Reply:
map.js is called after lengedary, I'll let it there for now.